jest-config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  testEnvironment: "jsdom",
```

Comment:
Do we want a single config or split configs for Jest + Native vs Jest + web (jsdom)?